src/main/java/com/pnudev/springonlinedelivery/MenuItemOrder.java

```diff
@@ -0,0 +1,30 @@
+package com.pnudev.springonlinedelivery;
+
+import java.util.Set;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.Getter;
+
+@Entity
+@Table(name = "MenuItemOrder")
+@AllArgsConstructor
+@NoArgsConstructor
+@Setter
+@Getter
+public class MenuItemOrder {
```

Comment:
You do not need to create additional entities to handle relations or for intermediate tables, all of this should be done using only JPA annotations (`@OneToMany`/`@ManyToMany` etc). Refer to https://www.baeldung.com/hibernate-one-to-many for info

Reply:
I removed the intermediate entity, fixed the relations (although I'm not sure if it's good). I need to decide what to do with the number of servings field